ballerina/Module.md

```diff
@@ -135,4 +135,19 @@ twitter:TweetCreateResponse postTweet = check twitter->/tweets.post( payload = {
 
 ```bash
 bal run
-```
\ No newline at end of file
+```
+
+
+**Note**: Obtained Access tokens are not explicitly expired. Follow the below step to invalidate access token
+
+### Invalidate access token
+
+    ```bash
+        curl --location --request POST "https://api.twitter.com/2/oauth2/revoke" ^
+        --header "Content-Type: application/x-www-form-urlencoded" ^
+        --data-urlencode "token=<your_access_token>" ^
+        --data-urlencode "client_id=<your_client_Id>" ^
+        --data-urlencode "token_type_hint=access_token"
+    ```
+
+    Replace `<your_client_id>`, and `<your_access_token>` with your specific values.
```

Comment:
Since we have created our setup guide without using the `offline.access` scope, it should expire in 2 hours so we don't need to mention this step specifically right? Or am I missing something?

Reply:
Yes, I removed that part and offline_access from the scope.